doc/md/versioned-migrations.mdx

```diff
@@ -55,6 +68,7 @@ import (
 
 func main() {
 	err := entc.Generate("./schema", &gen.Config{
+        //highlight-next-line
```

Comment:
didnt know i could do this!

Reply:
https://github.com/ent/ent/blob/master/doc/website/docusaurus.config.js#L89-L91

;)